docs/Configuration/yaml-files.md

```diff
@@ -338,7 +338,9 @@ software:
 
 - `app_store_id` is the ID of the Apple App Store app. You can find this at the end of the app's App Store URL. For example, "Bear - Markdown Notes" URL is "https://apps.apple.com/us/app/bear-markdown-notes/id1016366447" and the `app_store_id` is `1016366447`.
 
-> Make sure to include only the ID itself, and not the `id` prefix shown in the URL. The ID must be wrapped in quotes as shown in the example so that it is processed as a string. 
+> Make sure to include only the ID itself, and not the `id` prefix shown in the URL. The ID must be wrapped in quotes as shown in the example so that it is processed as a string.
+
+For App Store apps that are supported on iOS and/or iPadOS, self-service is set to false. For example, if the app is supported on macOS, iOS, and iPadOS, and self_service is set to true, it will only be applied for macOS.
```

Comment:
```suggestion
For App Store apps that are supported on iOS and/or iPadOS, `self_service` is set to `false`. For example, if the app is supported on macOS, iOS, and iPadOS, and `self_service` is set to `true`, it will only be applied for macOS.
```